### DISCUSSION TITLE
Run specified cop with specified Enforced Style?

### DISCUSSION THREAD

**Discussion by zw963** [rubocop/rubocop#11600] [Q&A]
e.g. I have a task, need convert all single quote string into double quotes for port Ruby code to Crystal easier.

so, i run rubocop like this.

```
rubocop -a --only Style/StringLiterals
```

But this command use the default behavior,  `EnforcedStyle: single_quotes`.

How can i specify it use `EnforcedStyle: double_quotes`?

For now, i have to create a specified config file, then run like: `rubocop -a --only Style/StringLiterals -c config_file` to archieve this.

Thanks.

**Comment by airled** [comment]
Seems like there is no option to pass enforced style as parameter. So I think using the config is the only option.
